src/libraries/System.Security.Cryptography.Pkcs/src/System/Security/Cryptography/Pkcs/CmsSignature.DSA.cs

```diff
@@ -48,17 +47,18 @@ internal override bool VerifySignature(
                 byte[] valueHash,
                 byte[] signature,
 #endif
-                string? digestAlgorithmOid,
-                HashAlgorithmName digestAlgorithmName,
+                Oid digestAlgorithm,
```

Comment:
Since all uses of this Oid value just use `.Value`, it seems like it should go back to being `digestAlgorithmOid`